src/Adjudications/AdjudicationHistory/AdjudicationHistoryTable.js

```diff
@@ -0,0 +1,104 @@
+import ResultsFilter from '../../Components/ResultsFilter'
+
+export const AdjudicationHistoryTable = ({
+  results,
+  totalRecords,
+  pageSize,
+  pageNumber,
+  changePerPage,
+  changePage,
+}) => (
+  <>
+    <ResultsFilter perPage={pageSize} pageNumber={pageNumber} totalResults={totalRecords} noBorder>
+      <ResultsFilter.PerPageDropdown handleChange={changePerPage} totalResults={totalRecords} perPage={pageSize} />
+    </ResultsFilter>
+
+    <LargeScreenOnly>
+      <Table
+        className="results"
+        head={
+          <Table.Row>
+            <Table.CellHeader setWidth="10%">Adjudication number</Table.CellHeader>
+            <Table.CellHeader setWidth="15%">Report date and time</Table.CellHeader>
+            <Table.CellHeader setWidth="20%">Establishment</Table.CellHeader>
+            <Table.CellHeader setWidth="45%">Offence description</Table.CellHeader>
+            <Table.CellHeader setWidth="10%">Finding</Table.CellHeader>
+          </Table.Row>
+        }
+      >
+        {results.map(result => (
+          <Table.Row key={result.adjudicationNumber}>
+            <Table.Cell>{result.adjudicationNumber}</Table.Cell>
+            <Table.Cell>{`${result.reportDate} ${result.reportTime}`}</Table.Cell>
+            <Table.Cell>{result.establishment}</Table.Cell>
+            <Table.Cell>{result.offenceDescription}</Table.Cell>
+            <Table.Cell>{result.findingDescription}</Table.Cell>
+          </Table.Row>
+        ))}
+      </Table>
+    </LargeScreenOnly>
+    <SmallScreenOnly>
+      <Table
+        className="results"
+        head={
+          <Table.Row>
+            <Table.CellHeader>Adjudication Number</Table.CellHeader>
+            <Table.CellHeader>Report date</Table.CellHeader>
+            <Table.CellHeader>Finding</Table.CellHeader>
+          </Table.Row>
+        }
+      >
+        {results.map(result => (
+          <Table.Row key={result.adjudicationNumber}>
+            <Table.Cell>{result.adjudicationNumber}</Table.Cell>
+            <Table.Cell>{result.reportDate}</Table.Cell>
+            <Table.Cell>{result.findingDescription}</Table.Cell>
+          </Table.Row>
+        ))}
+      </Table>
+    </SmallScreenOnly>
+
+    <div>
+      <PreviousNextNavigation
+        pagination={{ perPage: pageSize, pageNumber }}
+        totalRecords={totalRecords}
+        pageAction={changePage}
+      />
+    </div>
+  </>
+)
+
+AdjudicationHistoryTable.propTypes = {
+  totalRecords: PropTypes.number.isRequired,
+  pageSize: PropTypes.number.isRequired,
+  pageNumber: PropTypes.number.isRequired,
+  results: PropTypes.arrayOf(
+    PropTypes.shape({
+      adjudicationNumber: PropTypes.number.isRequired,
+      reportDate: PropTypes.string.isRequired,
+      reportTime: PropTypes.string.isRequired,
+      establishment: PropTypes.string.isRequired,
+      offenceDescription: PropTypes.string.isRequired,
+      findingDescription: PropTypes.string,
+    }).isRequired
```

Comment:
Is the `.isRequired` in the right place?

Reply:
I've moved this